FizzBuzz/FizzBuzz.rb

```diff
@@ -0,0 +1,11 @@
+def fizzBuzz(num)
+  if num % 3 == 0 && num % 5 == 0
+    puts "FizzBuzz"
+  elsif num % 3 == 0
+    puts "Fizz"
+  elsif num % 5 == 0
+    puts "Buzz"
+  else
+    puts num
```

Comment:
How about returning the value, rather than `puts`ing it? That way the calling function can decide what it wants to do with the result.